pkgs/games/openjk/default.nix

```diff
@@ -0,0 +1,38 @@
+{ stdenv, fetchFromGitHub, makeWrapper, cmake, libjpeg, zlib, libpng, mesa_noglu, SDL2 }:
+
+stdenv.mkDerivation rec {
+  name = "OpenJK-2017-08-11";
+
+  src = fetchFromGitHub {
+    owner = "JACoders";
+    repo = "OpenJK";
+    rev = "a0828f06e0181c62e110f2f78d30acb5036b4113";
+    sha256 = "1wbb643z2nyhyirzzy3rz03wjqglwmsgnj7w5cl8167f9f9j9w0m";
+  };
+
+  dontAddPrefix = true;
+  enableParallelBuilding = true;
+
+  nativeBuildInputs = [ makeWrapper cmake ];
+  buildInputs = [ libjpeg zlib libpng mesa_noglu SDL2 ];
+
+  preConfigure = ''
+    cmakeFlags="-DCMAKE_INSTALL_PREFIX=$out/opt"
```

Comment:
@Mic92 Fixed, thanks! BTW do you know a solution for this issue? https://github.com/NixOS/nixpkgs/pull/27096#issuecomment-324060858